.github/release_plan.md

```diff
@@ -24,6 +24,7 @@
 -   [ ] Update the [`release` branch](https://github.com/microsoft/vscode-jupyter/branches)
     -   [ ] If there are `release` branches that are two versions old (e.g. release-2020.[current month - 2]) you can delete them at this time
 -   [ ] Update `main` after creating the release branch. (Warning: this should happen right after creating the release branch. If this is deferred till later, the `main` and `release` branches can diverge significantly, which may cause merge conflicts.)
+    -   [ ] Bump the engines.vscode version on the `main` branch. For example, from `1.58.0-insider` to `1.59.0-insider`
```

Comment:
@DonJayamanne and I had a chat on the afternoon, instead of turning the daily insiders release on and off, we can leave it like that and update the engine at this point. This way stable won't get the next month's version early and we can keep the version's clearly.

Downside is we loose the daily insiders for a week. But that happens anyway if we disable it.

Reply:
Wouldn't this also bork the tests on main for one week?